guacamole/src/main/webapp/app/client/controllers/clientController.js

```diff
@@ -522,6 +545,18 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
             }
         }
 
+        // If one of the End keys is pressed, and we have a one keysym from each
+        // of Ctrl and Alt groups, send Ctrl-Alt-Delete.
+        if (END_KEYS[keysym] &&
+            !_.isEmpty(_.pick(ALT_KEYS, currentKeysPressedKeys)) &&
+            !_.isEmpty(_.pick(CTRL_KEYS, currentKeysPressedKeys))
+        ) {
+                event.preventDefault();
```

Comment:
Why is the body of this `if` indented by two levels?

Reply:
Ummm....variety is the spice of code?

Let me fix that real quick...